Branch-TestBed/Pods/Headers/Private/Nocilla/ASIHTTPRequestStub.h

```diff
@@ -0,0 +1 @@
+../../../Nocilla/Nocilla/Hooks/ASIHTTPRequest/ASIHTTPRequestStub.h
```

Comment:
I'm of the opinion that the Pods directory shouldn't be checked in. I'll leave that open ended, but I don't really think it should be.